README.md

```diff
@@ -145,16 +145,15 @@ Build a Go server that receives log messages in the format defined in `proto/log
     1. Add a binary target from `server.go`
         - use [`go_binary()`][go_binary]
         - name it `go-server`
-        - it depends on `server_lib`
-        - add `//proto/logger:logger_go_proto` to `deps`
```

Comment:
Is the dependency not needed?

Reply:
it built without and it is already a transitive dependency via the `go_library` rule.

Reply:
[the solution](https://github.com/tweag/nix_bazel_codelab/blob/philipp/solutions/go/cmd/server/BUILD.bazel#L18-L22) does not include this dependency either.

Reply:
Cool, sounds good then.